content/posts/10-year-anniversary.md

```diff
@@ -0,0 +1,65 @@
+Based on that success and the stress that came with it, we [created a company](https://www.ericholscher.com/blog/2014/oct/24/announcing-read-the-docs-for-business/) to focus on sustainability in 2014.
+We've been working on Read the Docs as a project for 14 years, and a company for 10 years.
+This post will reflect on the last 10 years of trying to make open source sustainable.
+
+## Our sustainability model
+
+We tried a number of different ways to make a sustainable open source project, really a service, over the years.
+The model that we've found that has worked for us currently is a version of the classic open source model used by GitHub, Travis CI, and others:
+
+* We offer a free service (Read the Docs Community) for open source projects, supported by a single non-tracking ad on the documentation.
+* We offer a paid service (Read the Docs Business) for companies that need private repos and additional benefits that make sense for larger organizations.
+
+We tried relying on donations and other optional support, but that didn't work at all, and left us struggling with [mental health issues](https://ericholscher.com/blog/2018/feb/7/the-post-i-never-published/) and burnout.
+We tried a model where we focused on documentation consulting and services, but that mostly took time away from actually working on the core product that open source projects rely on.
+We also [experimented with grant funding](https://blog.readthedocs.com/czi-grant-announcement/),
+which was a much more aligned source of funding allowing work on the core product,
+but is a one-time source of funding.
+
+Our goal is to work on the core product each day,
+and [our current model aligns the incentives](https://ericholscher.com/blog/2016/aug/31/funding-oss-marketing-money/) so that everyone who is using the product is supporting it in some way,
+and we can focus on making it the best it can be for all our users.
+
+## Lessons learned
+
+A few of the most important lessons from the last 10 years:
+
+* You don't get extra points for being open source or bootstrapped. People will compare you to proprietary, venture-backed services that have raised many millions of dollars. **We compete by focusing on documentation, a niche too small for most venture-backed businesses**. Venture-backed companies have to keep adding unrelated features in order to grow into other areas, and we can focus on our core competency.
+* Keeping trust in the community is the most important thing, because we're nothing without our users. It's a cliche, but trust is hard to earn and easy to lose. We continually think about how to balance sustainability with our the values of our users, and have made many decisions over the years about having "enough". **We can't maximize profit and continue to keep the trust of our community.**
```

Comment:
```suggestion
* Keeping trust in the community is the most important thing, because we're nothing without our users. It's a cliche, but trust is hard to earn and easy to lose. We continually think about how to balance sustainability with the values of our users, and have made many decisions over the years about having "enough". **We can't maximize profit and continue to keep the trust of our community.**
```